tobac/analysis.py

```diff
@@ -227,6 +371,46 @@ def cog_cell(
 def lifetime_histogram(
     Track, bin_edges=np.arange(0, 200, 20), density=False, return_values=False
 ):
+    """Compute the lifetime histogram of linked features.
+
+    Parameters
+    ----------
+    Track : pandas.DataFrame
+        Dataframe of linked features, containing the columns 'cell'
+        and 'time_cell'.
+
+    bin_edges : int or ndarray, optional
+        If bin_edges is an int, it defines the number of equal-width
+        bins in the given range. If bins is a ndarray, it defines a
+        monotonically increasing array of bin edges, including the
+        rightmost edge. Default is np.arange(0, 200, 20).
+
+    density : bool, optional
+        If False, the result will contain the number of samples in
+        each bin. If True, the result is the value of the probability
+        density function at the bin, normalized such that the integral
+        over the range is 1. Default is False.
+
+    return_values : bool, optional
+        Bool determining wether the lifetimes of the features are
+        returned from this function. Default is False.
+
+    Returns
+    -------
+    hist : ndarray
+        The values of the histogram.
+
+    bin_edges : ndarray
+        The edges of the histogram.
+
+    bin_centers : ndarray
+        The centers of the histogram intervalls.
+
+    minutes, optional : ndarray
```

Comment:
I think it would be better here to specify which parameter controls whether this appears or not.